### PR TITLE
Review PR for deviceexplorer-iothubexplorer.md

### PR DESCRIPTION
**Pull request by dominicbetts**
# Guidelines for reviewers
Using this Pull Request (PR), you can:

- Add general comments about the article in this PR
- Add a comment to a specific line in one of the files that make up this article
- Propose edits to the article

You can view the published version of this article at:
https://azure.microsoft.com/documentation/articles/deviceexplorer-iothubexplorer/

## Add general comments
To add a general comment, use this page (**Conversation** tab)

## Add comments to a specific line
To add a comment to specific line, go to the **Files changed** tab, hover your mouse pointer over the line in the file and click the blue **+** to add a new line comment.

## Propose edits to the article
To propose edits to this article, go to the **Files changed** tab, find the file you want to edit and click the pencil icon (**Change this file using the online editor**), and make your changes. When you are done you will be prompted to create a PR that contains your proposed changes.

> Note: The PR with your proposed changes will be pushing changes to the dominicbetts/azure-content-pr:edit-deviceexplorer-iothubexplorer branch.

### CODE REVIEW

File: doc/manage_iot_hub.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Manage IoT Hub
+
+Before a device can communicate with IoT Hub, you must add details of that device to the IoT Hub device identity registry. When you add a device to your IoT Hub device identity registry, the hub generates the connection string that the device must use when it establishes its secure connection to your hub. You can also use the device identity registry to disable a device and prevent it from connecting to your hub.
+
+To add devices to your IoT hub and manage those devices, you can use either of:
+
+- The cross-platform, command-line [iothub-explorer](#iothub-explorer) tool
+- The Windows-only, graphical [Device Explorer](#device-explorer) tool
+
+Use either of these tools to generate a device specific connection-string that you can copy and paste in the source code of the application running on your device. Both tools are available in this [repository][lnk-this-repo].
```
Comment (sethmanheim) [inline doc/manage_iot_hub.md]: No dash in "connection string."

File: doc/manage_iot_hub.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+npm install -g iothub-explorer
+```
+
+See [Install iothub-explorer][lnk-install-iothub-explorer] for more information.
+
+To provision a new device:
+
+1. Get the connection string for your IoT hub. See [Set up IoT Hub][setup-iothub] for more details.
+
+2.  Run the following command to register your device with your IoT hub. When you run the command, replace `<iothub-connection-string>` with the IoT Hub connection-string from the previous step and replace `<device-name>` with a name for your device such as **mydevice**.
```
Comment (sethmanheim) [inline doc/manage_iot_hub.md]: No dash in connection string.

File: doc/manage_iot_hub.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+2.  On the **Management** tab, click **Create** to register a new device with your IoT hub. The **Create Device** dialog appears. In the **Device ID** field, type a unique name for your device such as **mydevice**, or select **Auto Generate ID** to generate a unique ID. Then click **Create**.
+
+    ![][img-getstarted3]
+
+3.  The **Device Created** window appears, indicating that your device has been successfully registered with this IoT Hub.
+
+    ![][img-getstarted4]
+
+4. Right-click on a selected device to retrieve the connection-string for your device:
```
Comment (sethmanheim) [inline doc/manage_iot_hub.md]: Remove dash from connection string.

File: doc/manage_iot_hub.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+    ![][img-connstr]
+  
+5. Select **Copy connection string** to copy the device connection-string to the clipboard. You can now paste this connection-string into the source code of the device application you are working with. The samples in this repository use connection strings in the format `HostName=<iothub-name>.azure-devices.net;DeviceId=<device-name>;SharedAccessKey=<device-key>`.
```
Comment (sethmanheim) [inline doc/manage_iot_hub.md]: Remove dash from connection string.

File: doc/manage_iot_hub.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
+# Manage IoT Hub
```
Comment (sethmanheim) [inline doc/manage_iot_hub.md]: A general comment: I can't make comments on screen shots, but in the past people have complained about us showing SAS tokens. I realize they are probably expired, but we might want to check on recommended guidelines...

File: tools/iothub-explorer/readme.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+You can use the following command to get additional help about all the iothub-explorer commands and their parameters:
+
+```shell
+$ iothub-explorer help
+```
+
+<a name="identityregistry"/>
+## Working with the device identity registry
+
+Supply your IoT Hub connection-string once using the **login** command. This means you do not need to supply the connection-string for subsequent commands for the duration of the session (defaults to one hour):
```
Comment (sethmanheim) [inline tools/iothub-explorer/readme.md]: Remove dash from connection string.

File: tools/iothub-explorer/readme.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+Event received: 
+{ deviceId: 'myFirstDevice', windSpeed: 13.557703581638634 }
+
+Event received: 
+{ deviceId: 'myFirstDevice', windSpeed: 11.123057782649994 }
+```
+
+> Note: The **monitor-events** command requires you to provide the IoT Hub connection-string even if you have created a session using the **login** command.
+
+Use the following commands to send a cloud-to-device command and then wait for the device to respond with an acknowledgemnt:
```
Comment (sethmanheim) [inline tools/iothub-explorer/readme.md]: Typo: should be acknowledgment.

File: doc/manage_iot_hub.md (hunk 1 above)
Comment (dominicbetts) [inline doc/manage_iot_hub.md]: Done

File: doc/manage_iot_hub.md (hunk 2 above)
Comment (dominicbetts) [inline doc/manage_iot_hub.md]: Done

File: doc/manage_iot_hub.md (hunk 3 above)
Comment (dominicbetts) [inline doc/manage_iot_hub.md]: Done

File: doc/manage_iot_hub.md (hunk 4 above)
Comment (dominicbetts) [inline doc/manage_iot_hub.md]: Done

File: doc/manage_iot_hub.md (hunk 5 above)
Comment (dominicbetts) [inline doc/manage_iot_hub.md]: I'll check this.

File: tools/iothub-explorer/readme.md (hunk 6 above)
Comment (dominicbetts) [inline tools/iothub-explorer/readme.md]: Done

File: tools/iothub-explorer/readme.md (hunk 7 above)
Comment (dominicbetts) [inline tools/iothub-explorer/readme.md]: Done

**Comment by dominicbetts** [comment]
Processed all feedback up to here

**Pull request closed without merging**